src/arrayMethodJoin.js

```diff
@@ -4,8 +4,26 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let stringFromArray = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (i === this.length - 1 && typeof this[i] !== 'undefined') {
+        stringFromArray += `${this[i]}`;
```

Comment:
Well done, but no need to wrap element in backticks If adding it to string. You can just use  `stringFromArray += this[i]`.